src/shared/components/lazyMobXTable/LazyMobXTable.tsx

```diff
@@ -208,29 +213,33 @@ class LazyMobXTableStore<T> {
     @computed public get downloadData()
```

Comment:
@onursumer @alisman What are your thought on changing this function this way? Should we override this function in the `MutationTable` and `CNATableComponent`? Or should we pass some function for downloading? I think eventually we want to be able to support multiple download formats from mutation table, e.g. MAF/VCF/TXT. But for now maybe better to support this proposed MAF-like format with one row per sample.